MapboxDirections/MBRouteStep.swift

```diff
@@ -371,6 +371,60 @@ public enum ManeuverDirection: Int, CustomStringConvertible {
     }
 }
 
+extension String {
+    internal func tagValuesSeparatedByString(separator: String) -> [String] {
+        return componentsSeparatedByString(separator).map { $0.stringByTrimmingCharactersInSet(.whitespaceCharacterSet()) }.filter { !$0.isEmpty }
+    }
+}
+
+/**
+ Encapsulates all the information about a road.
+ */
+struct Road {
+    let names: [String]?
+    let codes: [String]?
+    let destinations: [String]?
+    let destinationCodes: [String]?
+    
+    init(name: String?, ref: String?, destination: String?) {
+        var codes: [String]?
+        if let names = name, let ref = ref ?? destination {
```

Comment:
`ref` is a misnomer here: what I really meant is “whatever is inside the parentheses”, which can be either a ref or a destination, due to munging that either the Directions API or OSRM performs (not sure which).